tests/app/internal/test_rest.py

```diff
@@ -0,0 +1,77 @@
+from flask import url_for
+
+import pytest
+
+
+@pytest.mark.parametrize(
+    'query_string',
+    [
+        ({'foo': 'bar'}, 'foo=bar'),
+        ({}, ''),
+        ({'foo': 'bar', 'baz': 'qux'}, 'foo=bar&baz=qux'),
+    ],
+)
+def test_it_get_internal(client, mocker, query_string):
```

Comment:
Are these technically unit (ut) tests since the endpoint calls no database or another method? tomayto tomahto 🍅 ? I don't think this needs to necessarily be change.

Reply:
These are unit tests, not integration tests - good catch. please change `it` to `ut`